pyanaconda/modules/storage/partitioning/manual/manual_module.py

```diff
@@ -162,15 +167,65 @@ def _create_request_for_device(self, device):
         :return: an instance of MountPointRequest
         """
         request = MountPointRequest()
-        request.device_spec = device.path
+        request.device_spec = self._btrfs_device_spec(device) or device.path
         request.format_type = device.format.type or ""
         request.reformat = False
+        request.mount_options += self._btrfs_mount_options(device)
 
         if device.format.mountable and device.format.mountpoint:
             request.mount_point = device.format.mountpoint
 
         return request
 
+    def _btrfs_mount_options(self, device):
+        """Get btrfs subvolume mount options.
+
+        :param device: a Blivet's device object
+        :return: a btrfs subvolume mount options
+        """
+        if device.raw_device.type == "btrfs subvolume":
+            return "subvol={}".format(device.name)
+        else:
+            return ""
+
+    def _btrfs_device_spec(self, device):
+        """Get btrfs device specification of the device.
+
+        Examples:
+        'UUID=2252ec30-1fce-4f8e-bdef-c50c3a44ede4'
+        'UUID=2252ec30-1fce-4f8e-bdef-c50c3a44ede4@root'
```

Comment:
Using `@` as delimiter here might be problematic -- naming subvolumes like `@home` is a common practice.

Reply:
Sure, that was a silly first stab, let's discuss it after we decide if we want it at all (device specification string for a btrfs subvolume).